src/parser.js

```diff
@@ -29,16 +37,39 @@ module.exports = {
                 rows.push(row);
             })
             .on('end', () => {
-                fs.rename(file, processedFile, (err) => {
-                    if (err) { return; }
 
-                    fs.writeFile(outputFile, JSON.stringify(rows, null, 2), (err) => {
-                        if (err) { return; }
+                // Rename was breaking when trying to perform operation in container
+                // Switched to CopyFileSync and got rid of callback
+                // Switched to try catch because no callback for copyfilesync
+                try {
+                    fs.copyFileSync(file, processedFile);
```

Comment:
if you copy it you need to unlink it when you're done. Renaming actually moves it.